<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>603</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Camera</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="CMS::ClickableLabel" name="frameLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>700</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>No image</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetButton">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <widget class="QTabWidget" name="tabWidget_2">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>0</y>
          <width>651</width>
          <height>491</height>
         </rect>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>General</string>
         </attribute>
         <widget class="QGroupBox" name="groupBox">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>0</y>
            <width>461</width>
            <height>451</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>Settings</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Clicking</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="1" column="0">
               <widget class="QLabel" name="label_8">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="sizeIncrement">
                 <size>
                  <width>1</width>
                  <height>1</height>
                 </size>
                </property>
                <property name="text">
                 <string>Dwell time</string>
                </property>
                <property name="scaledContents">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QDoubleSpinBox" name="dwellSpinBox">
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="maximum">
                 <double>5.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item row="2" column="0" colspan="5">
               <widget class="QSlider" name="dwellSlider">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximum">
                 <number>5000</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::NoTicks</enum>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QLabel" name="label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>s</string>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="0" colspan="3">
               <widget class="QCheckBox" name="enableClickingCheckBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="sizeIncrement">
                 <size>
                  <width>1</width>
                  <height>1</height>
                 </size>
                </property>
                <property name="text">
                 <string>Enable clicking</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Mouse Movement</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="4" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Vertical Gain</string>
                </property>
               </widget>
              </item>
              <item row="4" column="2">
               <widget class="QPushButton" name="lockGainButton">
                <property name="maximumSize">
                 <size>
                  <width>18</width>
                  <height>18</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="icons.qrc">
                  <normaloff>:/icons/images/lock-unlocked.png</normaloff>
                  <normalon>:/icons/images/lock-locked.png</normalon>:/icons/images/lock-unlocked.png</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="0" colspan="2">
               <widget class="QSlider" name="horizontalGainSlider">
                <property name="minimum">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <number>18</number>
                </property>
                <property name="singleStep">
                 <number>3</number>
                </property>
                <property name="value">
                 <number>6</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TicksAbove</enum>
                </property>
                <property name="tickInterval">
                 <number>3</number>
                </property>
               </widget>
              </item>
              <item row="0" column="0" colspan="2">
               <widget class="QCheckBox" name="reverseHorizontalCheckBox">
                <property name="text">
                 <string>Reverse Horizontal</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>High</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QCheckBox" name="smoothingCheckBox">
                <property name="text">
                 <string>Smoothing</string>
                </property>
               </widget>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Low</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>High</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Low</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Low</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0" colspan="2">
               <widget class="QSlider" name="verticalGainSlider">
                <property name="minimum">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <number>18</number>
                </property>
                <property name="singleStep">
                 <number>3</number>
                </property>
                <property name="value">
                 <number>6</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="invertedAppearance">
                 <bool>false</bool>
                </property>
                <property name="invertedControls">
                 <bool>false</bool>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TicksAbove</enum>
                </property>
                <property name="tickInterval">
                 <number>3</number>
                </property>
               </widget>
              </item>
              <item row="9" column="0" colspan="2">
               <widget class="QSlider" name="smoothingSlider">
                <property name="minimum">
                 <number>5</number>
                </property>
                <property name="maximum">
                 <number>95</number>
                </property>
                <property name="singleStep">
                 <number>5</number>
                </property>
                <property name="value">
                 <number>65</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="invertedAppearance">
                 <bool>true</bool>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TicksAbove</enum>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Horizontal Gain</string>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>High</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="autoDetectNoseCheckBox">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Auto Detect Nose</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>5-4-3-2-1</string>
         </attribute>
         <widget class="QWidget" name="verticalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>120</y>
            <width>481</width>
            <height>211</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QCheckBox" name="resetCheckF5">
             <property name="text">
              <string>Run the 5-4-3-2-1 routine on F5 key press</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>5</number>
             </property>
             <item>
              <widget class="QCheckBox" name="resetCheckTime">
               <property name="text">
                <string>Run the 5-4-3-2-1 routine automatically every</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="resetTime">
               <property name="text">
                <string>3</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="resetTimeUnit">
               <item>
                <property name="text">
                 <string>minutes</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>seconds</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="F5label">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="resetCheckLoss">
             <property name="text">
              <string>Run the 5-4-3-2-1 routine on loss of trackpoint</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>30</y>
            <width>531</width>
            <height>71</height>
           </rect>
          </property>
          <property name="text">
           <string>The 5-4-3-2-1 routine is a way to change the point being tracked on the face without clicking the image</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>969</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDevices">
    <property name="title">
     <string>Devices</string>
    </property>
   </widget>
   <addaction name="menuDevices"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CMS::ClickableLabel</class>
   <extends>QLabel</extends>
   <header>ClickableLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
